botocore/handlers.py

```diff
@@ -104,6 +104,13 @@ def check_for_200_error(response, **kwargs):
 
 def _looks_like_special_case_error(http_response):
     if http_response.status_code == 200:
+<<<<<<< HEAD
```

Comment:
Looks like an artefact from a bad merge / rebase

Reply:
Indeed! Thanks for pointing out!
Feeling foolish for letting that happen. Anyhow, fixed - please review again, thanks!